core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/environment/EnvironmentV4Endpoint.java

```diff
@@ -14,20 +14,20 @@
 
 import com.sequenceiq.cloudbreak.api.endpoint.v4.environment.requests.DatalakePrerequisiteV4Request;
```

Comment:
Are these imports going to be moved under Environment service as well?

Reply:
If it should go to the Environment service, then yes/